app/views/publish_data/monthly_service_metrics/_field_errors.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if field_has_errors(metrics, field) %>
+  <ul class="error-message">
+    <% metric_errors(metrics).each do |k, v| %>
+      <% v.each do |value| %>
+        <li><%= "#{k.to_s.humanize}: #{value}" if k == field %><li>
```

Comment:
I think for the field errors, we should remove the ```#{k.to_s.humanize}:``` and replace it with ```This``` so it reads ```This should be ....```.

Reply:
👍